test/t/test_sudo.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 
+from conftest import assert_bash_exec
 from conftest import assert_complete
```

Comment:
```suggestion
from conftest import assert_complete, assert_bash_exec
```

We have never put them in separate lines in other tests. It is good to keep it consistent.